src/models/user.js

```diff
@@ -119,26 +119,26 @@ const userSchema = new mongoose.Schema(
     },
     phoneNumber: {
       type: Number,
-      required: true,
-      unique: true,
+      // required: true,
+      // unique: true,
     },
     age: {
       type: Number,
-      required: true,
+      // required: true,
     },
     gender: {
       type: String,
-      required: true,
+      // required: true,
       enum: constants.ENUM_GENDER,
     },
     nationality: {
       type: String,
-      required: true,
+      // required: true,
```

Comment:
lets not have commented code, you can just delete all the commented code in this file